bin/i-pi-tests

```diff
@@ -0,0 +1 @@
+../tools/py/tests.py
```

Comment:
is this the most general way to link these files? why not making it a link to tests.py instead?

Reply:
I did a symbolic link  (ln -s ) like it is done for all the files in the bin folder except for "i-pi"
What are you suggesting?

Reply:
ah is this how a symbolic link appears in github? interesting, I did not know! sorry, I only looked at the review interface and not the command line. then you did what I suggested.